site/index.html

```diff
@@ -2,7 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Home</title>
+    <meta name="robots" content="noindex" />
+    <title>KPT - Home</title>

```

Comment:
we should use small cap `kpt` consistently everywhere.

Reply:
SGTM, done.